package.json

```diff
@@ -82,15 +86,13 @@
     "fork-ts-checker-webpack-plugin": "6.0.8",
     "husky": "4.3.8",
     "jest": "26.6.3",
-    "json-serverless": "1.6.14",
```

Comment:
Nerveless wasn't being used for this projects S3 capabilities so I removed them but fixed them in my stash. Needs some refactoring though so will be in a future PR